src/codewhisperer/service/transformationResultsViewProvider.ts

```diff
@@ -286,10 +287,14 @@ export class ProposedTransformationExplorer {
                     codeTransformJobId: transformByQState.getJobId(),
                     codeTransformApiErrorMessage: e?.message || errorMessage,
                     codeTransformRequestId: e?.requestId,
+                    result: MetadataResult.Fail,
+                    reason: 'ExportResultArchiveFailed',
                 })
                 throw new ToolkitError(errorMessage)
             }
 
+            const exportResultsArchiveSize = (await fs.promises.stat(pathToArchive)).size
```

Comment:
Not a blocker: This is the size of the zip folder. I believe for IntelliJ we are logging the byte size pre-zipped.

It will still help us determine overall size of upload/download

Reply:
What we could do is have IntelliJ measure zip size to match VSCode. Since looping through the files and calling `fs.stat` is the reason we did not implement this.

Reply:
Aligned offline, will use zip size.